types/hand-crafted/modules/xmpp/JingleSessionPC.d.ts

```diff
@@ -0,0 +1,49 @@
+import JingleSession from './JingleSession';
+import XmppConnection from './XmppConnection';
+import JitsiLocalTrack from '../RTC/JitsiLocalTrack';
+import CodecMimeType from '../../service/RTC/CodecMimeType';
```

Comment:
I think this needs to be:
```javascript
import { CodecMimeType } from '../../service/RTC/CodecMimeType';
```

Reply:
Thought I'd got all of those - I switched from types to const enums as they provide better semantics :-) I blame VS code for not picking up the error :-)